guides/common/modules/proc_installing-webhooks-plugin.adoc

```diff
@@ -12,8 +12,17 @@ After installing webhooks, you can configure {ProjectServer} to send webhook req
 # {foreman-installer} --enable-foreman-plugin-webhooks
 ----
 * Optionally, you can install the CLI plugin using the following command:
+
+** For {EL} 8:
```

Comment:
There is no difference in instructions between EL 7 & 8: the installer takes care of all of that. Please align on only using the installer for both EL 7 & 8.